Source/WebKit/GPUProcess/webrtc/LibWebRTCCodecsProxy.mm

```diff
@@ -88,15 +88,15 @@
 {
     protectedConnection()->removeWorkQueueMessageReceiver(Messages::LibWebRTCCodecsProxy::messageReceiverName());
 
-    protectedWorkQueue()->dispatch([this, protectedThis = WTFMove(refFromConnection)] {
-        assertIsCurrent(workQueue());
-        auto decoders = WTFMove(m_decoders);
+    protectedWorkQueue()->dispatch([protectedThis = WTFMove(refFromConnection)] {
+        assertIsCurrent(protectedThis->workQueue());
+        auto decoders = WTFMove(protectedThis->m_decoders);
```

Comment:
I guess we should use std::exchange(protectedThis->m_decoders, { }) instead of WTFMove here and below.